census_summary lint

From `main` GH action run:

```
tools/cell_census_builder/scripts/release/census_summary.py:10: error: Incompatible types in assignment (expression has type "None", variable has type "int")  [assignment]
tools/cell_census_builder/scripts/release/census_summary.py:11: error: Incompatible types in assignment (expression has type "None", variable has type "int")  [assignment]
tools/cell_census_builder/scripts/release/census_summary.py:13: error: Incompatible types in assignment (expression has type "bool", variable has type "str")  [assignment]
```

--- a/tools/cell_census_builder/scripts/release/census_summary.py
+++ b/tools/cell_census_builder/scripts/release/census_summary.py
@@ -7,10 +7,10 @@ import pandas as pd
 from tools.cell_census_builder.globals import CENSUS_DATA_NAME, CENSUS_INFO_NAME
 
 # Print all of the Pandas DataFrames, except the dimensions
-pd.options.display.max_columns = None
-pd.options.display.max_rows = None
+pd.options.display.max_columns = None  # type: ignore[assignment] # None is legal per Pandas documentation.
+pd.options.display.max_rows = None  # type: ignore[assignment] # None is legal per Pandas documentation.
 pd.options.display.width = 1024
-pd.options.display.show_dimensions = False
+pd.options.display.show_dimensions = False  # type: ignore[assignment] # boolean is legal per Pandas documentation.
 
 
 def display_summary(census_version: str) -> int:

